demos/form-control/Slider.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Ui\Form\Control;
+
+use Atk4\Ui\View;
+use Atk4\Ui\Js\JsExpression;
+use Atk4\Ui\Js\JsFunction;
+use Atk4\Ui\Js\JsReload;
+
+class Slider extends Input
```

Comment:
enabled/disabled/readonly demo should be added into `demos/form-control/input2.php`

and there should be probably a separate demo for Slicer + Behat tests are needed